nailgun/entities.py

```diff
@@ -2702,6 +2702,8 @@ def create_missing(self):
                  |-> domain
                  '-> environment
 
+        If nested entities contain only `id` (usually it means that there is no
+        `organization` field) perform additional read request.
```

Comment:
Didn't get your point in brackets. `(usually it means that entity was only initialized and not read)` maybe?